mmdet/apis/ote/extension/datasets/data_utils.py

```diff
@@ -295,3 +296,45 @@ def create_gt_box(x1, y1, x2, y2, label_name):
         dataset_items.append(dataset_item)
 
     return dataset_items
+
+
+def get_sizes_from_DatasetEntity(dataset: DatasetEntity, target_wh: list):
+    """
+    Function to get sizes of instances in DatasetEntity and to resize it to the target size. 
+
+    :param dataset: DatasetEntity in which to get statistics
+    :param target_wh: target width and height of the dataset
+    :return list: tuples with width and height of each instance
+    """
+    wh_stats = []
+    for item in dataset:
+        for ann in item.annotation_scene.annotations:
```

Comment:
Please be cautious about the rois. Not sure if those are handled properly this way.

Reply:
I believe it should be `item.get_annotations()`.

Reply:
Great